src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Migrations/OpenIdMigrations.cs

```diff
@@ -12,13 +12,13 @@ public int Create()
                 .Column<string>(nameof(OpenIdApplicationIndex.ApplicationId), column => column.WithLength(48))
                 .Column<string>(nameof(OpenIdApplicationIndex.ClientId), column => column.Unique()));
 
-            SchemaBuilder.CreateReduceIndexTable(nameof(OpenIdApplicationByPostLogoutRedirectUriIndex), table => table
-                .Column<string>(nameof(OpenIdApplicationByPostLogoutRedirectUriIndex.PostLogoutRedirectUri))
-                .Column<int>(nameof(OpenIdApplicationByPostLogoutRedirectUriIndex.Count)));
+            SchemaBuilder.CreateReduceIndexTable(nameof(OpenIdAppPostLogoutRedirectIndex), table => table
+                .Column<string>(nameof(OpenIdAppPostLogoutRedirectIndex.LogoutRedirectUri))
+                .Column<int>(nameof(OpenIdAppPostLogoutRedirectIndex.Count)));
 
             SchemaBuilder.CreateReduceIndexTable(nameof(OpenIdApplicationByRedirectUriIndex), table => table
```

Comment:
Should we rename the other indexes for consistency?

Reply:
Ok will do.